src/api/spec/models/watched_item_spec.rb

```diff
@@ -0,0 +1,176 @@
+require 'rails_helper'
+
+RSpec.describe WatchedItem do
```

Comment:
With [shoulda-matchers](https://github.com/thoughtbot/shoulda-matchers#usage) (we have this gem in our Gemfile), we can replace a lot of these specs with a single spec using the matcher `validate_uniqueness_of`.